web_dashboard_tile/tile.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+       <record model="ir.ui.view" id="dashboard_tile_tile_tree_view">
+            <field name="name">tile.tile.tree</field>
+            <field name="model">tile.tile</field>
+            <field name="type">tree</field>
```

Comment:
I don't know why runbot fails. (I personaly don't use it) 
I just checked log and saw that three warnings...
Please, yes remove this 3 lines. (It will be done, because it's an obsolete field).